src/docs/doxygen/GeosxConfig.hpp

```diff
@@ -114,10 +114,10 @@
 /* #undef chai_VERSION */
 
 /// Version information for adiak
-#define adiak_VERSION ..
+/* #undef adiak_VERSION */
```

Comment:
Please rollback this file. I know it's generated automatically... Sorry for this.

Reply:
Have'nt see it. should avoit git commit -a